applications/portal/values-idfint.yaml

```diff
@@ -5,8 +5,8 @@ config:
   volumes:
     sharedWorkarea:
       nfs:
-        path: "/share1/home/firefly/shared-workarea"
-        server: "10.22.240.130"
+        path: "/firefly-share/firefly/shared-workarea"
```

Comment:
Could you remind me why the path is slightly different than `idfdev` (`/firefly/`)

Reply:
I need to check that it actually is....

Reply:
Turns out it is int that doesn't match reality.